e2e-tests/src/config.rs

```diff
@@ -20,4 +20,8 @@ pub struct Config {
     /// seed value of sudo account
     #[clap(long)]
     pub sudo: Option<String>,
+
+    /// whether to print debug storage info
+    #[clap(long)]
+    pub storage_debug: bool,
```

Comment:
this is really confusing, because you can either run tests xor debug storage; so it should be rather a subcommand than just a flag which completely changes behavior and invalidates more of the other flags

however, what is even more confusing for me is that binary for running tests has an additional role of debugging storage, which definitely should be a part of our `cliain` tool; this would require including `cliain` for the e2e pipeline, but we would gain better organization imho

also, I would like to call out to @Marcin-Radecki for his opinion

Reply:
Yes, I think using `cliain` here would be a better fit. All the calls from this PR can be embedded into `cliain`, but it's more work for sure. For now, we can add a `Subcommand` `mode` for example which have two values `storage_debug` and `run_tests`, with the latter as default. This is ofc just a suggestion :)

Reply:
added this to cliain